landing page/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>TedX Nit Kurukshetra</title>
```

Comment:
Missing meta-tags. Tags like author, links, description, charset, etc are missing. 
Check the source-code of [techspardha.org](http://techspardha.org/) for more information.